components/table/stories/template.js

```diff
@@ -322,6 +330,7 @@ export const Template = ({
 					${when(hasMenu || isSortable, () => html`
 						${when(isSortable, () => Button({
 								size: "m",
+								variant: "secondary",
```

Comment:
@rise-erpelding I don't mind changing this to the secondary button, but I feel like I still needed all of the passthroughs anyways. I think I can remove the `--mod-button-height`, alongside a `min-block-size` to help out the funky alignment we were seeing, but that felt like that was all 😞